src/ducks/SANCharts/ChartSidecar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Icon from '@santiment-network/ui/Icon'
+import RecentlyWatched from '../../components/RecentlyWatched/RecentlyWatched'
+import styles from './ChartSidecar.module.scss'
+
+const ChartSidecar = () => {
+  return (
+    <div className={styles.wrapper}>
+      <Icon type='arrow-left' className={styles.toggle} />
+      <div className={styles.content}>
+        <div className={styles.visible}>
+          <RecentlyWatched
+            className={styles.recents}
+            onProjectClick={console.log}
```

Comment:
is it wip? @DmitriVanGuard

Reply:
![image](https://user-images.githubusercontent.com/25135650/60538894-3116df00-9d14-11e9-9937-97c01f45c299.png)